code/game/gamemodes/cult/talisman.dm

```diff
@@ -278,11 +250,104 @@ Rite of Disorientation
 	user.equip_to_slot_or_del(new /obj/item/clothing/head/culthood/alt(user), slot_head)
 	user.equip_to_slot_or_del(new /obj/item/clothing/suit/cultrobes/alt(user), slot_wear_suit)
 	user.equip_to_slot_or_del(new /obj/item/clothing/shoes/cult/alt(user), slot_shoes)
 	user.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	user.drop_item()
 	user.put_in_hands(new /obj/item/weapon/melee/cultblade(user))
+	user.put_in_hands(new /obj/item/weapon/restraints/legcuffs/bola/tactical(user))
+
+/obj/item/weapon/paper/talisman/armor/attack(mob/living/target, mob/living/user)
+	if(iscultist(user) && iscultist(target))
+		user.drop_item()
+		invoke(target)
+		qdel(src)
+		return
+	..()
+
+//Talisman of Horrors: Breaks the mind of the victim with nightmarish hallucinations
+/obj/item/weapon/paper/talisman/horror
+	cultist_name = "Talisman of Horrors"
+	cultist_desc = "A talisman that will break the mind of the victim with nightmarish hallucinations."
+	color = "#ffb366" // light orange
+	invocation = "Lo'Nab Na'Dm!"
+
+/obj/item/weapon/paper/talisman/horror/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		user.visible_message("<span class='cultitalic'>You disturb [target] with visons of the end!</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/H = target
+			H.reagents.add_reagent("mindbreaker", 30)
+		qdel(src)
+
+//Talisman of Fabrication: Creates a construct shell out of 25 metal sheets.
+/obj/item/weapon/paper/talisman/construction
+	cultist_name = "Talisman of Construction"
+	cultist_desc = "Use this talisman on at least twenty-five metal sheets to create an empty construct shell"
+	invocation = "Ethra p'ni dedol!"
+	color = "#000000" // black
+
+/obj/item/weapon/paper/talisman/construction/attack_self(mob/living/user)
+	if(iscultist(user))
+		user << "<span class='warning'>To use this talisman, attack the target directly.</span>"
+	else
+		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
+	
+	
+/obj/item/weapon/paper/talisman/construction/attack(obj/M,mob/living/user)
+	if(iscultist(user))
+		user << "<span class='cultitalic'>This talisman will only work on a stack of metal sheets!</span>"
+		log_game("Construct talisman failed - not a valid target")
+
+//Talisman of Shackling: Applies special cuffs directly from the talisman
+/obj/item/weapon/paper/talisman/shackle
+	cultist_name = "Talisman of Shackling"
+	cultist_desc = "Use this talisman on a victim to handcuff them with dark bindings."
+	invocation = "In'totum Lig'abis!"
+	color = "#B27300" // burnt-orange
+	uses = 4
+
+/obj/item/weapon/paper/talisman/shackle/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		if(isrobot(target))
+			..()
+			return
+		if(!isliving(target))
+			user.visible_message("<span class='cultitalic'>This talisman's magic does not affect the dead!</span>")
+			return
+		var/mob/living/L = target
```

Comment:
This is unneeded, as the proc argument `mob/living/target` already typecasts the target as mob/living